### PR TITLE
docs: add metadata fields for all tags

### PR DESCRIPTION
**Pull request by pseudoyu**

### CODE REVIEW

File: versions/v1.0.0/Metaverse/Trade/Sell.adoc [hunk 1]
```diff
@@ -27,6 +27,25 @@ interface Sell {
...
+`name` *Optional*:: Name of the sold metaverse token, e.g. `Beanz`.
+`standard` *Optional*:: Standard of the sold metaverse token, e.g. `ERC-721`.
+`symbol` *Optional*:: Symbol of the sold metaverse token, e.g. `BEANZ`.
+`value` *Required*:: Amount of the sold metaverse token, e.g. `1`.
+`uri` *Optional*:: URI of the sold metaverse token, e.g. `ipfs://QmdYeDpkVZedk1mkGodjNmF35UNxwafhFLVvsHrWgJoz6A/beanz_metadata/159`.
+`parsed_image_url` *Optional*:: Parsed image URL of the sold metaverse token, e.g. `https://ipfs.io/ipfs/QmdYeDpkVZedk1mkGodjNmF35UNxwafhFLVvsHrWgJoz6A/beanz_metadata/159`.
+`cost` *Required*:: Cost of the sold metaverse token. Should include the following fields.
+`cost.name` *Optional*:: Name of the cost token, e.g. `Wrapped Ether`.
+`cost.symbol` *Optional*:: Symbol of the cost token, e.g. `WETH`.
+`cost.decimals` *Optional*:: Decimal places of the cost token, e.g. `18`.
```
Comment (FrankLi123) [inline versions/v1.0.0/Metaverse/Trade/Sell.adoc:44]: Different terms to describe the same concepts ( e.g., decimals of the cost token  and decimal places of the cost token)

File: versions/v1.0.0/Social/Comment/main.adoc [hunk 2]
```diff
@@ -39,6 +39,35 @@ interface Comment {
...
 ----
 
+===== Fields
+
+`handle` *Optional*:: Handle/username of the user who is commenting, e.g. `looktanl.lens`.
+`body` *Required*:: Main content of the comment, e.g. `Cool.🌸`.
+`title` *Optional*:: Title of the comment.
+`summary` *Optional*:: Short summary of the comment.
+`media` *Optional*:: Array of media objects that are attached to the comment.
+`media.address` *Required*:: Url/address of the media object.
```
Comment (FrankLi123) [inline versions/v1.0.0/Social/Comment/main.adoc:49]: Inconsistency in the capitalization of URLs (Url and URL)

File: versions/v1.0.0/Social/Comment/main.adoc [hunk 3]
```diff
@@ -39,6 +39,35 @@ interface Comment {
...
+`author_url` *Optional*:: Url of the commentor.
+`profile_id` *Optional*:: Profile ID of the commentor, e.g. `0x01ed62`.
+`publication_id` *Optional*:: Publication ID of the comment, e.g. `0x1d`.
+`content_uri` *Optional*:: Content URI of the comment, e.g. `ar://TN8jBlQja25mMgyZIDGoiMAjctKUneUyLyKdY9ZdCnE`.
+`timestamp` *Optional*:: Timestamp of the comment, e.g. `1712293392`.
+`target` *Required*:: Post or comment being commented.
+`target.handle` *Optional*:: Handle/username of the user who created the original post or comment, e.g. `yoginth.lens`.
+`target.body` *Required*:: Main content of the original post or comment, e.g. `Just dropped a song about @lens/hey, mind's blown 🤯 Made with AI.`.
+`target.title` *Optional*:: Title of the original post or comment.
+`target.summary` *Optional*:: Short summary of the original post or comment.
```
Comment (FrankLi123) [inline versions/v1.0.0/Social/Comment/main.adoc:61]: Redundant word 'Short'

File: versions/v1.0.0/Social/Reward/main.adoc [hunk 4]
```diff
@@ -34,6 +34,31 @@ interface Reward {
...
+`content_uri` *Optional*:: Content URI of the post.
+`timestamp` *Optional*:: Timestamp of the post.
+`reward` *Required*:: Reward for specific post. Should include the following fields.
+`reward.address` *Optional*:: Address of the reward token, If the token is a native token, this field is not required, e.g. `0xc2132D05D31c914a87C6611C10748AEb04B58e8F`.
+`reward.id` *Optional*:: ID of the reward token.
+`reward.value` *Required*:: Amount of the reward, e.g. `690000`.
+`reward.name` *Optional*:: Name of the reward token, e.g. `(PoS) Tether USD`.
+`reward.symbol` *Optional*:: Symbol of the reward token, e.g. `USDT`.
+`reward.uri` *Optional*:: URI of the reward token.
+`reward.decimals` *Optional*:: Decimal of the reward token, e.g. `6`.
```
Comment (FrankLi123) [inline versions/v1.0.0/Social/Reward/main.adoc:59]: Forgot to +s      |   inconsistency in using Decimal and Decimal places

File: versions/v1.0.0/Social/Share/main.adoc [hunk 5]
```diff
@@ -39,6 +39,35 @@ interface Share {
...
 
+===== Fields
+
+`handle` *Optional*:: Handle/username of the user who is sharing, e.g. `rabby1.lens`.
+`body` *Optional*:: Main content of the post.
+`title` *Optional*:: Title of the post.
+`summary` *Optional*:: Short summary of the post.
+`media` *Optional*:: Array of media objects that are attached to the post.
+`media.address` *Required*:: Url/address of the media object.
+`media.mime_type` *Required*:: Mime type of the media object.
```
Comment (FrankLi123) [inline versions/v1.0.0/Social/Share/main.adoc:50]: Is there a need to explain the meaning of mime type? What is it?

File: versions/v1.0.0/Collectible/Transfer/main.adoc [hunk 6]
```diff
@@ -18,6 +18,17 @@ interface Transfer {
...
+===== Fields
+
+`address` *Optional*:: Address of the transferred collectible token, e.g. `0x5452C7fB99D99fAb3Cc1875E9DA9829Cb50F7A13`.
+`id` *Optional*:: ID of the transferred collectible token, e.g. `1032`.
+`name` *Optional*:: Name of the transferred collectible token, e.g. `The Genesis RSS3 Avatar NFT`.
+`standard` *Optional*:: Standard of the transferred collectible token, e.g. `ERC-721`.
+`symbol` *Optional*:: Symbol of the transferred collectible token, e.g. `The Genesis RSS3 Avatar NFT`.
+`value` *Required*:: Amount/value of the transferred collectible token, e.g. `1`.
+`uri` *Optional*:: URI of the transferred collectible token, e.g. `ipfs://QmdHTECGDf8u4vNe4PLXSqBTBkYqxiCaYYfdJiUhtgiBKR/6`.
+`parsed_image_url` *Optional*:: The parsed image URL of the transferred collectible token, e.g. `https://ipfs.io/ipfs/QmSX9QiwjTGbk5m22UscTg3vrbMwUfFsmxVzMI57hkPD5U/1032.png`.
```
Comment (FrankLi123) [inline versions/v1.0.0/Collectible/Transfer/main.adoc:30]: Redundant word 'The', conflicting with previous descriptions.

File: versions/v1.0.0/Exchange/Staking/Stake.adoc [hunk 7]
```diff
@@ -23,6 +23,21 @@ interface Stake {
...
+`action` *Required*:: Action of the staking transaction, e.g. `stake`.
+`tokens` *Required*:: Array of tokens staked from the staking pool.
+`tokens.address` *Optional*:: Contract address of the staked token.
+If the token is a native token, this field is not required, e.g. `0xc98D64DA73a6616c42117b582e832812e7B8D57F`.
+`tokens.value` *Required*:: Amount of the staked token, e.g. `5764251642130434218884`.
+`tokens.name` *Optional*:: Name of the staked token, e.g. `RSS3`.
+`tokens.symbol` *Optional*:: Symbol of the staked token, e.g. `RSS3`.
+`tokens.decimals` *Optional*:: Decimals of the staked token, e.g. `18`.
+`tokens.standard` *Optional*:: Standard of the staked token, e.g. `ERC-20`.
+`period` *Optional*:: Period of the staking, if the staking has a time locker.
```
Comment (FrankLi123) [inline versions/v1.0.0/Exchange/Staking/Stake.adoc:37]: Could add round brackets here (if the staking has a time locker)

File: versions/v1.0.0/Social/Delete/main.adoc [hunk 8]
```diff
@@ -24,6 +24,22 @@ interface Delete {
 }
 ----
 
+===== Fields
+
+`handle` *Optional*:: Handle/username of the user who deleted the post, e.g. `ordi.csb`.
```
Comment (FrankLi123) [inline versions/v1.0.0/Social/Delete/main.adoc:29]: What does Handle/username mean? why is handle used as an alternative to username?

File: versions/v1.0.0/Social/Profile/Update.adoc [hunk 9]
```diff
@@ -20,6 +20,19 @@ interface Update {
...
 ----
 
+===== Fields
+
+`action` *Required*:: Action of updating a profile, e.g. `update`.
+`profile_id` *Optional*:: Updated ID of the profile, e.g. `0x01766e`.
+`address` *Optional*:: Updated address of the profile, e.g. `0x76d4ffda8c66864fa07f0a285b8e01e11b4bb7b3`.
+`handle` *Optional*:: Updated handle/username of the profile, e.g. `brinse.lens`.
+`image_uri` *Optional*:: Updated image URI of the profile, e.g. `https://media.orb.ac/thumbnailDimension768/https://statics-polygon-lens.s3.eu-west-1.amazonaws.com/profile/nft-0x6441543C6537FF430f2e1d6fE8FFFcfc05d52D04_polygon_0x5D666F215a85B87Cb042D59662A7ecd2C8Cc44e6_3097622.gif`.
+`bio` *Optional*:: Updated biography of the profile.
```
Comment (FrankLi123) [inline versions/v1.0.0/Social/Profile/Update.adoc:30]: what does biography mean in this context?

File: versions/v1.0.0/Metaverse/Trade/Sell.adoc (hunk 1 above)
Comment (pseudoyu) [reply versions/v1.0.0/Metaverse/Trade/Sell.adoc:44]: It's defined in node project using same Token struct.

File: versions/v1.0.0/Social/Comment/main.adoc (hunk 2 above)
Comment (pseudoyu) [reply versions/v1.0.0/Social/Comment/main.adoc:49]: resolved.

File: versions/v1.0.0/Social/Comment/main.adoc (hunk 3 above)
Comment (pseudoyu) [reply versions/v1.0.0/Social/Comment/main.adoc:61]: resolved.

File: versions/v1.0.0/Social/Reward/main.adoc (hunk 4 above)
Comment (pseudoyu) [reply versions/v1.0.0/Social/Reward/main.adoc:59]: added.

File: versions/v1.0.0/Social/Share/main.adoc (hunk 5 above)
Comment (pseudoyu) [reply versions/v1.0.0/Social/Share/main.adoc:50]: a specific standard for media types, may be no need to explain in a protocol doc?

https://developer.mozilla.org/en-US/docs/Glossary/MIME_type

File: versions/v1.0.0/Collectible/Transfer/main.adoc (hunk 6 above)
Comment (pseudoyu) [reply versions/v1.0.0/Collectible/Transfer/main.adoc:30]: dropped.

File: versions/v1.0.0/Exchange/Staking/Stake.adoc (hunk 7 above)
Comment (pseudoyu) [reply versions/v1.0.0/Exchange/Staking/Stake.adoc:37]: revised.

File: versions/v1.0.0/Social/Delete/main.adoc (hunk 8 above)
Comment (pseudoyu) [reply versions/v1.0.0/Social/Delete/main.adoc:29]: `handle` is a specific username in some platform like lens, so we keep both.

File: versions/v1.0.0/Social/Profile/Update.adoc (hunk 9 above)
Comment (pseudoyu) [reply versions/v1.0.0/Social/Profile/Update.adoc:30]: The brief introduction of a person? Perhaps just using a bio is sufficient.

**Review by FrankLi123 — APPROVED** [review]

**Pull request merged**